include/quda_milc_interface.h

```diff
@@ -443,13 +441,7 @@ extern "C" {
    * @param clover_coeff Clover coefficient
    * @param source Right-hand side source field
    * @param solution Solution spinor field
-   * @param ritz_prec Precision of the ritz vectors (2 - double, 1 - single)
-   * @param vec_infile input file location 
-   * @param vec_outfile output file location
-   * @param max_search_dim eigCG parameter: search space dimention
-   * @param nev eigCG parameter: how many eigenpairs to compute within one eigCG call
-   * @param deflation_grid eigCG parameter : how many eigenpairs to compute within the incremental phase (# of eigenpairs = nev*deflation_grid)
-   * @param tol_restart initCG parameter : at what tolerance value to restart initCG solver
+   * @param eig_args contains info about deflation space
    * @param rhs_idx  bookkeep current rhs
    * @param last_rhs_flag  is this the last rhs to solve?
    * @param final_residual Array of true residuals
```

Comment:
There should be no space between doxygen comment blocks and the function they are commenting